packages/dataviews/src/dataform-controls/boolean.tsx

```diff
@@ -1,57 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { privateApis } from '@wordpress/components';
```

Comment:
Could we instead do the following?

- Remove the `boolean` form control. So `Edit: boolean` doesn't work, which is a breaking change we need to document in the changelog. Only `Edit: checkbox` or `Edit: toggle` are available.
- Update the `boolean` [field type](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/field-types/boolean.tsx#L51) to use `Edit: checkbox`.

Reply:
At the time boolean was added (https://github.com/WordPress/gutenberg/pull/70567) it probably mimicked existing controls such as text, integer, etc. I think it's best if we separate the field type (boolean) from the Edit, the controls that render it (checkbox, toggle).

Reply:
Additionally, for both `checkbox` and `toggle` controls we need to leverage the `field.description` to offer a description for the control ([example](https://github.com/WordPress/gutenberg/blob/90929734517ea7ea0c479ae56839600a3e4a10f7/packages/dataviews/src/dataform-controls/utils/validated-text.tsx#L73)).

Reply:
Hi @oandregal, thank you for the review all your feedback was applied:
<img width="273" height="273" alt="Screenshot 2025-09-04 at 16 48 20" src="https://github.com/user-attachments/assets/0ff92085-6306-4321-a507-4be53319e21c" />